pkgs/by-name/gr/graphene-hardened-malloc/package.nix

```diff
@@ -23,9 +23,17 @@ stdenv.mkDerivation (finalAttrs: {
   # these tests cover use as a build-time-linked library
   checkTarget = "test";
 
+  buildPhase = ''
+    runHook preBuild
+
+    for VARIANT in default light; do make VARIANT=$VARIANT; done
```

Comment:
```suggestion
    for VARIANT in default light; do make $makeFlags ''${enableParallelBuilding:+-j$NIX_BUILD_CORES} VARIANT=$VARIANT; done
```

Reply:
Done, also - what exactly is that '' doing here? Tried removing it and it didn't build anymore 😄

Reply:
ow, I messed up the force push

Reply:
> Done, also - what exactly is that '' doing here? Tried removing it and it didn't build anymore 😄

`${}` is string interpolation so it has to be escaped with `''`

Reply:
oh, right, lmao, for some reason I thought we wanted to do nix string interpolation here